src/dials/util/export_shelx.py

```diff
@@ -167,3 +170,36 @@ def _write_ins(experiment_list, best_unit_cell, ins_file):
                 )
             )
         LATT_SYMM(f, sg)
+        logger.info(
+            f"Using composition {composition} to write SFAC & UNIT instructions"
+        )
+        sorted_composition = _parse_compound(composition)
+        f.write("SFAC %s\n" % " ".join(sorted_composition))
+        f.write(
+            "UNIT %s\n"
+            % " ".join(
+                [str(sorted_composition[e] * sg.order_z()) for e in sorted_composition]
+            )
+        )
+        f.write("HKLF 4\n")  # Intensities
+        f.write("END\n")
+
+
+def _parse_compound(composition):
+    elements = chemical_elements.proper_caps_list()[:94]
+    m = re.findall(r"([A-Z][a-z]?)(\d*)\s*", composition)
```

Comment:
The trailing `\s*` in the regular expression is redundant in a `re.findall`.  This suggestion will parse `C6 H10 N4 O2` and `C6H10N4O2` equally well.
```suggestion
    m = re.findall(r"([A-Z][a-z]?)(\d*)", composition)
```

Reply:
@huwjenkins what do you think? I was wondering if you actually want to _enforce_ a space between items. Probably not, right?

Reply:
no that wasn't the intent. Happy for someone with better understanding like @benjaminhwilliams to push a commit to fix this :grin:

Reply:
I've committed @benjaminhwilliams's suggestion